docusaurus/docs/Android/04-compose/02-component-architecture.mdx

```diff
@@ -79,6 +81,7 @@ override fun onCreate(savedInstanceState: Bundle?) {
     setContent {
         ChatTheme { // Theme wrapper
             ChannelList(
+                modifier = Modifier.fillMaxSize(),
```

Comment:
Without this the component wraps `LoadingIndicator` or `EmptyContent` making it appear top-start instead of center

Reply:
Yep, noticed that as well 👍